src/Raven.Studio/typescript/models/resources/widgets/databaseNotificationsItem.ts

```diff
@@ -0,0 +1,83 @@
+class databaseNotificationsItem implements databaseAndNodeAwareStats {
+    database: string;
+    nodeTag: string;
+    
+    alertsCount: number;
+    alerts: Raven.Server.Dashboard.Cluster.Notifications.DatabaseNotifications.NotificationSummaryItem[];
+
+    performanceHintsCount: number;
+    performanceHints: Raven.Server.Dashboard.Cluster.Notifications.DatabaseNotifications.NotificationSummaryItem[];
+
+    noData: boolean;
+
+    hideDatabaseName: boolean;
```

Comment:
```suggestion
    hideDatabaseName = false
```

and remove `this.hideDatabaseName = false;` at 18 line